themes/custom_tmpl/templates/search/home.phtml

```diff
@@ -0,0 +1,46 @@
+<?
```

Comment:
I chose search/home because it doesn't change very often and because it's effectively the homepage of VuFind. If any changes happen here, they're likely to be in the rows on the bottom (customized out) or in the searchbox itself (separate template).